src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+interface TabsPropsType {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+}
+
+export const Tabs: React.FC<TabsPropsType>
+  = ({ tabs, selectedTabId, onTabSelected }) => {
+    const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+    const setSelectedTab = (tab: Tab) => {
+      if (tab.id !== selectedTabId) {
+        onTabSelected(tab);
+      }
+    };
+
+    return (
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => (
+              <li
+                key={tab.id}
+                className={classNames({
+                  'is-active': selectedTab.id === tab.id,
```

Comment:
you checked is current tab selected several times
will be better to use separate boolean variable and use it here and pass it in `setSelectedTab` function